Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN npm install --production
 
 COPY . /src
 
+HEALTHCHECK --interval=10s --timeout=10s --start-period=5s --retries=3 CMD node healthcheck.js || exit 1
```

Comment:
Why do we use different timings than in amass scanner?

(Same comment than amass PR:)
Timeout seems to be too long. I would suggest to use maybe 3 to 5 seconds. It should be at least shorter than the interval...

Reply:
all scanners now have the same 5s interval